### ISSUE TITLE
dockerignore is not working

### ISSUE THREAD

**Issue by alisinabh** [fandoghpaas/fandogh-cli#250]
# Issue

Files and directories specified in `.dockerignore` and `.fandoghignore` is not getting excluded from uploaded context.

## Description

As fandogh documentation suggests, adding a `.dockerignore` (or `.fandoghignore`) should prevent the specified entries from
getting uploaded to fandogh build servers, But as I have tested, Those files are getting added in the build context whatsoever.

## Version

```
# fandogh --version
fandogh, version 1.19.1
```

## How to reproduce?

```
mkdir test-fandogh
cd test-fandogh
# create a large file with random data
dd if=/dev/urandom of=largefile seek=0 count=102400
# add the file to .dockerignore entries
echo "/largefile" > .dockerignore
fandogh image init --name testimg
fandogh image publish --version v0.1
# you can see the build context getting large and fandogh-cli will complain about workspace size.
```

Thank you.

**Comment by ghost** [comment]
Hi @alisinabh, 

Thank you so much for raising this issue. We will address this problem soon. 
for now if you remove the / prefix from the address it should work as expected. 
e.g. echo "/largefile" > .dockerignore

**Comment by alisinabh** [comment]
Thank you @behroozmirzaie7

**Comment by s-soroosh** [comment]
Thank you @alisinabh for the report. 
The bug is fixed in version `1.21.0`.
